auth/grants.go

```diff
@@ -78,6 +78,8 @@ type ClaimGrants struct {
 	Metadata string `json:"metadata,omitempty"`
 	// Key/value attributes to attach to the participant
 	Attributes map[string]string `json:"attributes,omitempty"`
+	// if a participant can subscribe to metrics
+	CanSubscribeMetrics *bool `json:"canSubscribeMetrics,omitempty"`
```

Comment:
can we put this into `Video` grant? this is where the other permission bits are stored. not named the best, but seems more consistent given where things are today.

Reply:
okay, I put it here because of the naming.

Reply:
Moved it in this [commit](https://github.com/livekit/protocol/pull/795/commits/1add410bf995f5e8e69926accf3e517fd187f908). A few more things
- Grouped struct with its methods (makes it easier on my brain :-) )
- It defaults to `false`, so not adding to things like `BuildSIPToken`, `BuildAgentToken`.